src/components/pages/accounts/IncomeExpenseHistogram.tsx

```diff
@@ -15,7 +15,7 @@ export default function IncomeExpenseHistogram({
 }: IncomeExpenseHistogramProps): JSX.Element {
   const interval = Interval.fromDateTimes(
     selectedDate.minus({ months: 6 }).startOf('month'),
-    selectedDate.endOf('month'),
+    selectedDate,
```

Comment:
don't use endOf('month') so we can check quantities middle of the month in the histogram. A big plus is also that then the quantities in the histogram and in the accounts table do match